readthedocs/projects/tasks.py

```diff
@@ -573,6 +578,15 @@ def run_build(self, docker, record):
 
         if self.build_env.failed:
             self.send_notifications(self.version.pk, self.build['id'])
+            # send build failure status to git Status API
+            self.send_build_status(
+                self.build['id'], BUILD_STATUS_FAILURE
+            )
+        elif self.build_env.successful:
```

Comment:
What is the other state here? Should this just be an `else`? If not, we should do an `else` with a `log.warning`.

Reply:
@ericholscher there are no other states (`BuildEnvironment`) , it will succeed or fail (in the Build Environment) at this point of the build. we could add an warning if we want.